docs/developer/index.rst

```diff
@@ -521,24 +521,30 @@ made in the future).
 PHA Filtering
 -------------
 
-Filtering of a :py:class:`~sherpa.astro.data.DataPHA` object has two
+Filtering of a :py:class:`~sherpa.astro.data.DataPHA` object has four
 complications compared to :py:class:`~sherpa.data.Data1D` objects:
 
-- the independent axis can be referred to in channel units (normally 1
+* the independent axis can be referred to in channel units (normally 1
   to the maximum number of channels), energy units (e.g. 0.5 to 7
   keV), or wavelength units (e.g. 20 to 22 Angstroms);
 
-- each channel has a width of 1, so channel filters - which are
+* each channel has a width of 1, so channel filters - which are
   generally going to be integer values - map exactly, but each channel
   has a finite width in the derived units (that is, energy or
   wavelength) so multiple values will map to the same channel;
 
-- the data can be dynamically grouped via the
+* the data can be dynamically grouped via the
   :py:attr:`~sherpa.astro.data.DataPHA.grouping` attribute, normally set
   by methods like :py:meth:`~sherpa.astro.data.DataPHA.group_counts` and
   controlled by the :py:meth:`~sherpa.astro.data.DataPHA.group` method,
   which means that the desired filter, when mapped to channel units,
   is likely to end up partially overlapping the first and last groups,
+  which means that `notice(a, b)` and `ignore(None, a); ignore(b, None)`
```

Comment:
```suggestion
  which means that ``notice(a, b)`` and ``ignore(None, a); ignore(b, None)``
```
Double backticks will stop Sphinx from trying to resolve that into some kind of link.